app/assets/stylesheets/common/base/discourse.scss

```diff
@@ -424,7 +424,7 @@ table {
 }
 
 .do-not-disturb-background {
-  display: flex;
+  display: flex !important;
```

Comment:
Is the `!important` necessary? It actually looks like the `. do-not-disturb-background` element is removed above.